src/olympia/addons/tests/test_tasks.py

```diff
@@ -451,13 +451,14 @@ def test_flag_high_hotness_according_to_review_tier_threshold_check_negative():
         upper_adu_threshold=1000,
         growth_threshold_before_flagging=10,
     )
-    # Average hotness should be negative, -0.1.
+    # Average hotness should be negative, -0.2.
     # growth_threshold_before_flagging for that tier is 10 so we should flag
-    # add-ons with hotness above 0
+    # add-ons with hotness above -0.1, but we explicitly avoid flagging
+    # negative growth add-ons, so only the last one should be flagged.
 
-    addon_factory(average_daily_users=251, hotness=-0.1)
-    addon_factory(average_daily_users=251, hotness=-0.2)
-    addon = addon_factory(average_daily_users=251, hotness=0.0)
+    addon_factory(average_daily_users=251, hotness=-0.05)
+    addon_factory(average_daily_users=251, hotness=-0.65)
```

Comment:
We should still include one with hotness 0.0 to make sure it is clear how that growth would be handled.

Reply:
Added more tests in 44ae308e7a8a485a26530014298bec90053a0e34